docs/SQP/index.md

```diff
@@ -11,12 +11,12 @@ The Special Quality Profiles are Custom made and tested Quality Profile for spec
 
 ## The Following SQP are available
 
-- [Streaming Optimized (SQP-1)](/SQP/1){:target="_blank" rel="noopener noreferrer"}
-- [Streaming Optimized (SQP-1 2160p)](/SQP/1-4k){:target="_blank" rel="noopener noreferrer"}
-- [UHD Remux|Bluray|IMAX-E (SQP-2)](/SQP/2){:target="_blank" rel="noopener noreferrer"}
-- [UHD Remux|IMAX-E (SQP-3)](/SQP/3){:target="_blank" rel="noopener noreferrer"}
-- [UHD WEBDL|IMAX-E (SQP-4)](/SQP/4){:target="_blank" rel="noopener noreferrer"}
-- [UHD Bluray|IMAX-E (SQP-5)](/SQP/5){:target="_blank" rel="noopener noreferrer"}
+- [SQP-1 (1080p)|*Streaming Optimized*](/SQP/1){:target="_blank" rel="noopener noreferrer"}
```

Comment:
Maybe these should be formatted as "SQP-1 (1080p) - Streaming Optimized" i.e., use a hyphen separator instead of a pipe.

Reply:
sounds good to me

Reply:
Added.